build-aux/m4/bitcoin_qt.m4

```diff
@@ -331,7 +341,8 @@ AC_DEFUN([_BITCOIN_QT_CHECK_STATIC_LIBS], [
   elif test "x$TARGET_OS" = xdarwin; then
     PKG_CHECK_MODULES([QTCLIPBOARD], [Qt5ClipboardSupport${qt_lib_suffix}], [QT_LIBS="-lQt5ClipboardSupport${qt_lib_suffix} $QT_LIBS"])
     PKG_CHECK_MODULES([QTGRAPHICS], [Qt5GraphicsSupport${qt_lib_suffix}], [QT_LIBS="-lQt5GraphicsSupport${qt_lib_suffix} $QT_LIBS"])
-    PKG_CHECK_MODULES([QTCGL], [Qt5CglSupport${qt_lib_suffix}], [QT_LIBS="-lQt5CglSupport${qt_lib_suffix} $QT_LIBS"])
+  elif test "x$TARGET_OS" = xwindows; then
+    PKG_CHECK_MODULES([QTWINDOWSUIAUTOMATION], [Qt5WindowsUIAutomationSupport], [QT_LIBS="-lQt5WindowsUIAutomationSupport $QT_LIBS"])
```

Comment:
To handle `DEBUG=1` this line should be:
```suggestion
    PKG_CHECK_MODULES([QT_WINDOWSUIAUTOMATION], [Qt5WindowsUIAutomationSupport${qt_lib_suffix}], [QT_LIBS="$QT_WINDOWSUIAUTOMATION_LIBS $QT_LIBS"])
```

Instead of `$QT_WINDOWSUIAUTOMATION_LIBS` could be just `-lQt5WindowsUIAutomationSupport${qt_lib_suffix}`, but I prefer the former variant as I think it is more robust and maintainable.

Reply:
Left this as the later for now, we can refactor all the checks later on/as a part of supporting qt6.

Reply:
Agree.